lib/util/requestSources.ts

```diff
@@ -0,0 +1,9 @@
+export const REQUEST_SOURCES = [
```

Comment:
maybe we should put these in an SDK one day?

Reply:
agree. URA is missing couple sources, and would benefit from shared SDK